### ISSUE TITLE
Gravity script.

### ISSUE THREAD

**Issue by cpl** [dragosthealex/LudumDare36#2]
noGravityScript.velocity = ((m1 - m2) / (m1 + m2)) * v1 + ((2 * m2) / (m1 + m2)) * v2;

// DFQ

**Comment by dragosthealex** [comment]
Yeah, tried using the elastic collision formula from elementary physics. It didn't go too well
